libraries/chain/db_market.cpp

```diff
@@ -641,25 +641,16 @@ int database::match( const limit_order_object& bid, const call_order_object& ask
    // TODO remove when we're sure it's always false
    bool before_core_hardfork_342 = ( maint_time <= HARDFORK_CORE_342_TIME ); // better rounding
    // TODO remove when we're sure it's always false
```

Comment:
Since you have removed the line to which this comment refers, remove it as well.

Reply:
Done as part of #1743